src/Microsoft.Management.UI.Internal/Microsoft.PowerShell.GraphicalHost.csproj

```diff
@@ -1,11 +1,13 @@
-<Project Sdk="Microsoft.NET.Sdk.WindowsDesktop" ToolsVersion="15.0">
+<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0">
   <Import Project="..\..\PowerShell.Common.props" />
   <PropertyGroup>
     <Description>Assembly containing WPF code for Out-GridView, HelpWindow, and Show-Command</Description>
     <NoWarn>$(NoWarn);CS1570</NoWarn>
     <RootNamespace>Microsoft.Management.UI.Internal</RootNamespace>
     <AssemblyName>Microsoft.PowerShell.GraphicalHost</AssemblyName>
     <UseWPF>True</UseWPF>
+    <TargetPlatformIdentifier>Windows</TargetPlatformIdentifier>
+    <TargetPlatformVersion>7.0</TargetPlatformVersion>
```

Comment:
Just curious, what does the `TargetPlatformVersion` mean? Meaning targeting win-7?

Reply:
Yes, that is the OS version

Reply:
We might have to have separate Win7 and Win10 packages as I think it's required to set this to 10 to pull in the WinRT packages

Reply:
Can we try setting this to 10 in a private branch and see if that helps the WinRT issue?

Reply:
I'm doing a local build to verify this

Reply:
I think it is too late in the release to generate 2 packages for windows. We should do that for 7.2 if needed.